manifests/n/NetSarangComputer,Inc/Xftp8/8.0.0068/NetSarangComputer,Inc.Xftp8.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.9.14.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: NetSarangComputer,Inc.Xftp8
+PackageVersion: 8.0.0068
+InstallerType: exe
+InstallerSwitches:
+  Silent: Y
+  SilentWithProgress: Y

```

Comment:
These switches don't seem to work
![image](https://github.com/user-attachments/assets/3c289bfc-6913-4788-a38d-b40eeac56b3e)